main/boost/APKBUILD

```diff
@@ -38,89 +58,94 @@ _libs="date_time
 	wserialization
 	"
 for _lib in $_libs; do
+	echo "$_options_carch" | grep -F -q -e "--without-${_lib}" || \
```

Comment:
the `... | grep ...` here will make it fork. We don't want fork in global scope (if we do that in every APKBUILD we will have a performance issue when calculating build order).